src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncNodesRequestHandler.scala

```diff
@@ -77,7 +77,7 @@ class FastSyncNodesRequestHandler(
       val evm = n.getAccount.codeHash
       val storage = n.getAccount.storageRoot
 
-      nodesKeyValueStorage.put(n.hash, n.toBytes)
+      saveNodeFn(n.hash, n.toBytes)
```

Comment:
I think @LukasGasior1 addressed on the VSU that he will be working on an improvement that involves fixing this, so I would leave it as it is for now.